source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -242,6 +270,9 @@ class RouteSerializer(GenericSerializer):
     line = serpy.MethodField(display_none=False)
     stop_points = StopPointSerializer(many=True, display_none=False)
 
+    def get_line_jsonschema_pre_type_mapping(self):
```

Comment:
I think it's better to add an attribute to MethodField (but we either need to change serpy or to use a custom MethodField)